### ISSUE TITLE
DLA-34 model

### ISSUE THREAD

**Issue by sonctmirasoft** [see--/keras-centernet#7]
Hi, do you have any plan to implement the DLA-34 (deep layer aggregation) network described in the paper? I think it provide a better speed/accuracy trade off than the hourglass model (3 times faster for 3 to 4 less AP on COCO dataset).

**Comment by see--** [comment]
Hi @sonctmirasoft ,

yes, I looked into DLA models. The problem is that TensorFlow does not support deformable convolutions. As you read, all other networks need them. There are open source libraries for deformable convolutions, but they aren't optimized. I don't think that you get 3x speedup. The hourglass model is probably still faster.

**Comment by sonctmirasoft** [comment]
Thanks for the head up, I guess we have to use the Pytorch version for now.

**Comment by zyc4me** [comment]
@see-- do you have the plan to release the training code？

**Comment by bhack** [comment]
You can upvote here for deformable convolution v2 https://github.com/tensorflow/addons/issues/179
